pennylane/ops/qubit/parametric_ops.py

```diff
@@ -436,8 +436,8 @@ def compute_matrix(phi):  # pylint: disable=arguments-differ
         **Example**
 
         >>> qml.PhaseShift.compute_matrix(torch.tensor(0.5))
-        tensor([[0.9689-0.2474j, 0.0000+0.0000j],
-                [0.0000+0.0000j, 0.9689+0.2474j]])
+        tensor([[1.0000+0.0000j, 0.0000+0.0000j],
+                [0.0000+0.0000j, 0.8776+0.4794j]])
```

Comment:
Why this is changed?

Reply:
It was an error in the docstring, I was testing something with phase shift and I noticed it was different so I fixed it. This is the correct return for the above command.